src/noc/containers/noc-errors/noc-error.component.spec.ts

```diff
@@ -0,0 +1,36 @@
+import { SimpleChange, SimpleChanges } from '@angular/core';
+import { NocNavigationEvent } from '../../models';
+import { NocErrorComponent } from './noc-error.component';
+
+describe('NocErrorComponent', () => {
+    let component: NocErrorComponent;
+    let store: any;
+
+    beforeEach(() => {
+        store = jasmine.createSpyObj('store', ['pipe', 'dispatch']);
+        component = new NocErrorComponent(store);
+    });
+
+    it('ngOnChanges', () => {
+        const simpleChanges = {} as SimpleChanges;
+        simpleChanges.navEvent = {} as SimpleChange;
+        component.navEvent.event = NocNavigationEvent.CONTINUE;
+        component.ngOnChanges(simpleChanges);
+        component.navigationHandler = jasmine.createSpy('navigationHandler');
+        expect(store.pipe).toHaveBeenCalled();
+    });
+
+    it('navigationHandler CONTINUE', () => {
+        component.navigationHandler(NocNavigationEvent.BACK);
+        expect(store.dispatch).toHaveBeenCalled();
+    });
+
+    it('navigationHandler BACK', () => {
+        component.navigationHandler(NocNavigationEvent.CONTINUE);
+        expect(store.dispatch).not.toHaveBeenCalled();
+    });
+
+    it('navigationHandler SUBMIT', () => {
```

Comment:
None of the test titles in here have descriptive names - they should take the form of:
`should throw an error when trying to submit`

This explains what's been tested and what the expected outcome is. At the moment, the titles are _vaguely_ explaining what's being tested.

Reply:
Though this is not written by me but agreed, will make this change.

Reply:
This hasn't been changed.